tools/pd-heartbeat-bench/config-template.toml

```diff
@@ -1,14 +1,15 @@
 round = 0
 
 store-count = 100
+hot-store-count = 10
 region-count = 2000000
 
 replica = 3
 
 leader-update-ratio = 0.06
-epoch-update-ratio = 0.0
-space-update-ratio = 0.0
-flow-update-ratio = 0.0
+epoch-update-ratio = 0.04
+space-update-ratio = 0.15
+flow-update-ratio = 0.35
```

Comment:
Can you explain why it should be `0.04/0.15/0.35`?

Reply:
Copy from older configure